.github/workflows/contract-testing.yaml

```diff
@@ -701,3 +704,45 @@ jobs:
 
           make run-contract-testing
           make delete-test-resources
+
+  stream-instance:
+    needs: change-detection
+    if: ${{ needs.change-detection.outputs.stream-instance == 'true' }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: setup Atlas CLI
+        uses: andreaangiolillo/atlas-cli-github-action@fdc1bca49f3f965831572078f1587090c818419a
```

Comment:
could you please update this action as per https://github.com/mongodb/mongodbatlas-cloudformation-resources/pull/934 ?

Reply:
Done!